setup.py

```diff
@@ -195,8 +195,8 @@ def _get_arrow_lib_as_linker_input(self):
         'azure-common<2.0.0',
         'azure-storage-blob>=12.0.0,<13.0.0',
         'boto3>=1.4.4,<2.0.0',
+        # While requests is vendored, we use regular requests to perform OCSP checks
```

Comment:
Did we not decide to _not_ pin requests to a specific version?

Reply:
I pinned it to next major release just in case to be safe. I was thinking that we can always relax it later if we want to, but it's still pinned currently. However, there hasn't been many major releases for `requests`.

Do you feel like we could unpin it safely?